back/lib/tasks/single_use/20240213_migrate_analysis_main_custom_field.rake

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+namespace :migrate_analysis do
```

Comment:
@kogre It might be valuable to have your feedback on the migration script.

Reply:
Great remarks, I now initialize the `main_field_ids` to `Analysis::Analysis.pluck(:main_custom_field_id).compact` which should make the script more idempotent.